src/utils.js

```diff
@@ -27,21 +28,21 @@ const indexOf = function(list, elem) {
 /**
  * Return whether an element is contained in a list
  */
-const contains = function(list, elem) {
+const contains = function<T>(list: Array<T>, elem: T) {
```

Comment:
The return value for this could be `boolean`.

Reply:
Whoops. I wish flow would catch missing types.

Reply:
We could start using https://github.com/gajus/eslint-plugin-flowtype#eslint-plugin-flowtype-rules-require-return-type.

Reply:
Yes please!